src/main/java/ch/uzh/ifi/seal/soprafs20/entity/Game.java

```diff
@@ -51,25 +57,49 @@
     @ElementCollection
     private List<String> clues = new ArrayList<String>();
 
-    @Column(nullable = false)
-    private int timestamp;
+    @Column
+    @ElementCollection
+    private List<Integer> score = new ArrayList<>();
 
-    @Column(nullable = false)
+    @Column
+    private LocalTime timestamp;
+
+    @Column
     private int wordsGuessedCorrect;
 
-    @Column(nullable = false)
+    @Column
     private int wordsGuessedWrong;
 
-    @Column(nullable = false)
+    @Column
     private int cardStackCount;
 
-    @Column(nullable = false)
+    @Column
     private int cardGuessedCount;
 
+    @Column()
+    private CardStatus cardStatus;
+
     public Long getId() {
         return id;
     }
 
+
+    public List<Integer> getLastWordIndex() {
```

Comment:
Need the lastWordIndex to restrict the user from submitting same index as of last time in case of rejection